shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -176,6 +199,17 @@ - (instancetype)initWithViewController:(FlutterViewController*)viewController {
   return self;
 }
 
+- (BOOL)isFirstResponder {
+  if (!self.flutterViewController.viewLoaded || !self.flutterViewController.view.window) {
```

Comment:
The second half of this check isn't doing anything useful; if `self.flutterViewController.view.window` is nil, then `[self.flutterViewController.view.window firstResponder]` is also nil, and thus can't be equal to `self`.